python.d/ipfs.chart.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Description: IPFS netdata python.d module
+# Authors: Pawel Krupa (paulfantom), davidak
+
+from base import UrlService
+import json
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+# default job configuration (overridden by python.d.plugin)
+# config = {'local': {
+#     'update_every': update_every,
+#     'retries': retries,
+#     'priority': priority,
+#     'url': 'http://localhost:5001/'
+# }}
+
+# charts order (can be overridden if you want less charts, or different order)
+ORDER = ['bandwidth', 'peers']
+
+CHARTS = {
+    'bandwidth': {
+        'options': [None, 'IPFS Bandwidth', 'kbits/s', 'Bandwidth', 'ipfs.bandwidth', 'line'],
+        'lines': [
+            ["in", None, "absolute", 8, 1000],
+            ["out", None, "absolute", -8, 1000]
+        ]},
+    'peers': {
+        'options': [None, 'IPFS Peers', 'peers', 'Peers', 'ipfs.peers', 'line'],
+        'lines': [
+            ["peers", None, 'absolute']
+        ]}
+}
+
+
+class Service(UrlService):
+    def __init__(self, configuration=None, name=None):
+        UrlService.__init__(self, configuration=configuration, name=name)
+        if len(self.url) == 0:
+            self.baseurl = "http://localhost:5001/"
+        else:
+            self.baseurl = self.url
```

Comment:
Those lines won't have any meaning since `self.url` will be overridden when `python.d.plugin` executes `check` method (from UrlService).